mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/GEOR_custom.js

```diff
@@ -0,0 +1,80 @@
+Ext.namespace("GEOR");
+
+GEOR.custom = {
+    HEADER_HEIGHT: 90,
```

Comment:
Here we obviously miss the previous behavior where the header height was obtained from shared maven filters.
We need to talk about this one.